### PR TITLE
Turned off RSpec/Description cop for system specs.

### PR DESCRIPTION
**Pull request by jmax-fearless**

### CODE REVIEW

File: .default.yml [hunk 1]
```diff
@@ -77,10 +77,11 @@ Rails/SkipsModelValidations:
 # See also the list of RSpec statements that we exempt from the block
 # length cop, under Metrics/BlockLength above
 
-# For this group of specs, there is no class
+# For these specs, there is no class
 RSpec/DescribeClass:
   Exclude:
     - 'spec/lib/tasks/**'
+    - 'spec/lib/system/**'
```
Comment (MothOnMars) [inline .default.yml:84]: should be `'spec/system/**'`, not `'spec/lib/system/**'`

**Review by MothOnMars — CHANGES_REQUESTED** [review]

File: .default.yml (hunk 1 above)
Comment (jmax-fearless) [reply .default.yml:84]: Um. Thanks.

Comment (jmax-fearless) [reply .default.yml:84]: Fixed.

**Comment by MothOnMars** [comment]
Note: when merging, be sure to update this with the relevant JIRA ticket.

**Comment by jmax-fearless** [comment]
Will open a new one for this.
(Git tidiness).

**Pull request closed without merging**